### PR TITLE
Nsng 75 webumenia button

### PR DESCRIPTION
**Pull request by igor-kamil**
Snímka obrazovky 2022-11-06 o 0 07 36

detail:

Snímka obrazovky 2022-11-06 o 0 07 21

### CODE REVIEW

File: resources/js/components/WebumeniaButton.vue [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<template>
+    <a :href="props.url" target="_blank" class="bg-green p-2.5 rounded-xl block">
```
Comment (rastislav-chynoransky) [inline resources/js/components/WebumeniaButton.vue:2]:
```suggestion
    <a :href="props.url" target="_blank" class="bg-green border-2 border-green p-2.5 rounded-xl block">
```

**Review by rastislav-chynoransky — CHANGES_REQUESTED** [review]

File: resources/js/components/AuthoritySummary.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="flex">
+    <div class="flex items-center">
```
Comment (rastislav-chynoransky) [inline resources/js/components/AuthoritySummary.vue:2]: Good catch! I'd prefer to set `self-center` to the flex item at 6th line to keep the avatar top-aligned.

**Review by rastislav-chynoransky — APPROVED** [review]
Thanks for the changes!
Btw, will the wording of the button change? The capitalized words "Archiv" and "Umenia" seem a bit off...

**Comment by igor-kamil** [comment]
Yes. The wording will change. It's just temporary. But they can't come up with "proper" idea for weeks... I think this can "accelerate the process" 😅

**Pull request merged**